### PR TITLE
Minor: Updated docker image

### LINKED ISSUE #50: Set happy-bank-core docker image as container in core pod
Image is place: https://hub.docker.com/r/jmatejka/happy-bank-core/tags

### PR DESCRIPTION
**Pull request by Oleksandr6676**
Changed core-app pod docker image
Closes #50

### CODE REVIEW

**Review by JanMate — CHANGES_REQUESTED** [review]
As I mentioned in #43, fix this as the first. K8s logs should help you.

**Comment by Oleksandr6676** [comment]
> 
Hi @JanMate.  I was just surfing across our core app and noticed one thing. This is what I get when I'm trying to execute app.py file using cmd inside the pod
Снимок экрана (25)
r

**Comment by JanMate** [comment]
> > 
> 
> Hi @JanMate. I was just surfing across our core app and noticed one thing. This is what I get when I'm trying to execute app.py file using cli inside the pod Снимок экрана (25)

It works by the expectation. If you check entrypoint and command in Dockerfile of core app, you will see that you need to run poetry run. It's because of using python virtual env controlled by poetry to easy maintenance. Of course, it means you don't need to override neither command nor args in the spec of core pod. 

So try to run it via poetry run and you will see running core app.

File: kubernetes/core-pod.yml [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
   containers:
-  - name: busybox-container
-    image: busybox
+  - name: core-app-container
+    image: jmatejka/happy-bank-core:1.0.0
     args:
     - /bin/sh
     - -c
-    - touch tmp/healthy; sleep infinity;
+    - touch healthy; ls; sleep infinity;
```
Comment (JanMate) [inline kubernetes/core-pod.yml:14]: As I've already mentioned, this overrides 'command' property in Dockerfile, so please remove it.

File: kubernetes/core-pod.yml [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
       initialDelaySeconds: 15
       periodSeconds: 10
     livenessProbe:
       exec:
         command:
         - cat
-        - tmp/healthy
+        - healthy
       initialDelaySeconds: 20
       periodSeconds: 10
```
Comment (JanMate) [inline kubernetes/core-pod.yml:30]: Health checks could use http and REST api endpoint.

File: kubernetes/core-svc.yml [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 metadata:
-  name: busybox-svc  
+  name: core-app-svc  
   labels:
-    app: busybox
+    app: core-app
 spec:
   type: ClusterIP
   ports:
     - protocol: TCP
```
Comment (JanMate) [inline kubernetes/core-svc.yml:10]: `TCP` protocols smells to me when we want to use REST api. If you know what I mean.

**Review by JanMate — CHANGES_REQUESTED** [review]

File: kubernetes/core-pod.yml (hunk 2 above)
Comment (Oleksandr6676) [reply kubernetes/core-pod.yml:30]: Hi @JanMate. I've already added http health check with /health endpoint on my local branch. I just haven't pushed it yet. But thank you for the hint

File: kubernetes/core-svc.yml (hunk 3 above)
Comment (JanMate) [reply kubernetes/core-svc.yml:10]: Sorry, my bad. I didn't realized the protocol on the right level of TCP/IP model. Solved.

File: kubernetes/core-pod.yml [hunk 4]
```diff
@@ -1,30 +1,30 @@
...
-    image: busybox
+  - name: core-app-container
+    image: jmatejka/happy-bank-core:1.0.0
     args:
     - /bin/sh
     - -c
-    - touch tmp/healthy; sleep infinity;
+    - touch healthy; ls; sleep infinity;
     ports:
      - containerPort: 80
```
Comment (JanMate) [inline kubernetes/core-pod.yml:16]: What does the number of the container pod mean? What is it for? Is it correct for our core app microservice?

**Comment by JanMate** [comment]
Verified. It works properly when you set ports correctly. 😉

File: kubernetes/core-pod.yml (hunk 4 above)
Comment (Oleksandr6676) [reply kubernetes/core-pod.yml:16]: Thank you! As per documentation flask application uses port 5000 by default. Now it works!

**Review by JanMate — APPROVED** [review]

**Pull request merged**